test/helpers/deploy-helpers.ts

```diff
@@ -30,16 +30,15 @@ export const approveBulk = async (
   }
 };
 
-export const mintBulk = async (
-  signers : Array<SignerWithAddress>,
+export const transferBulk = async (
+  users : Array<SignerWithAddress>,
+  admin : SignerWithAddress,
   amount : bigint,
   zns : IZNSContractsLocal | IZNSContracts,
 ) => {
-  for (const signer of signers) {
-    await zns.meowToken.connect(signer).mint(
-      signer.address,
-      amount
-    );
+  for (const user of users) {
+    await zns.zToken.connect(admin).approve(await zns.treasury.getAddress(), ethers.MaxUint256);

```

Comment:
There is no need for this here. Think about what this call is doing, we're connecting as the `admin` and calling to approve the treasury. We wouldn't need to do this in a loop even if it was correct, but it also isn't needed here at all because we don't require approval just to do the below transfer from `admin` to `user`

Reply:
yep